### ISSUE TITLE
Repair olist_hestiaUI Component After core_hestiaUI Decided for Left-Aligned As Default

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#46]
# Description
```
Please provide a short description of what you have encountered below.
```
`oList_hestiaUI` Componnet requires a repair work after `core_hestiaUI` was adjusted to use left-alignment as default instead of its initial centered version.

# Expected Behavior
```
Please specify the expected behavior of your usage below.
```
`olist_hestiaUI` component is back to seamless usable component.

# Current Behavior
```
Please specify the current behavior of software below.
```
`olist_hestiaUI` component's UI design is broken.

# Steps to Reproduce (COMPULSORY)
```
Please specify all the steps required to re-produce the problem.
```

# Attachments
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
fixed in 69d42b8d487b1c9bf0bdcf15a12e636cf50a24c5

**Comment by hollowaykeanho** [comment]
Released in https://github.com/ZORALab/Hestia/releases/tag/v1.0.0
